app-modules/service-management/src/Filament/Resources/IncidentResource.php

```diff
@@ -67,13 +66,11 @@ public static function getRecordSubNavigation(Page $page): array
             EditIncident::class,
         ];
 
-        if (IncidentUpdateFeature::active()) {
-            $navigationItems = [
-                ViewIncident::class,
-                EditIncident::class,
-                ManageIncidentUpdate::class,
-            ];
-        }
+        $navigationItems = [
+            ViewIncident::class,
+            EditIncident::class,
+            ManageIncidentUpdate::class,
+        ];
```

Comment:
We should refactor this just to be inline in `generateNavigationItems` and remove the overwritten first definition of `$navigationItems`